crates/uv-python/src/installation.rs

```diff
@@ -350,7 +350,7 @@ impl Ord for PythonInstallationKey {
     fn cmp(&self, other: &Self) -> std::cmp::Ordering {
         self.implementation
             .cmp(&other.implementation)
-            .then_with(|| other.version().cmp(&self.version()))
+            .then_with(|| self.version().cmp(&other.version()))
```

Comment:
I think this is inverted.

Reply:
Sorry I should've added a explanation here. This is intentional to ensure that Python installations are sorted in descending order by version. https://github.com/astral-sh/uv/issues/5139

Reply:
Uh oh! haha

Reply:
Hah, I'll take a look.